addon/helpers/d3-area.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import { area } from 'd3-shape';
+
+export function d3Area( [ xScale, yScale ], { xKey, yKey } ) {
+  let xFn  = (d) => xScale( xKey( d ) );
+  let y1Fn = (d) => yScale( yKey( d ) );
+  let y0Fn = ()  => yScale( 0 );
```

Comment:
I'm looking into this as a possible bug. I have charts where the area overlaps the xAxis. I think y0Fn needs to be set to yMin for a single series chart.